README.md

```diff
@@ -1 +1,214 @@
+
+``` 
+echo "deb http://packages.matrix.one/matrix-creator/ ./" | sudo tee --append /etc/apt/sources.list
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install matrix-creator-init matrix-creator-malos cmake g++ git libzmq3-dev --no-install-recommends
+reboot
+```
+
+Add some dependencies and pocketsphinx package:
+
+``` 
+echo "deb http://unstable-packages.matrix.one/ stable main" | sudo tee -a /etc/apt/sources.list
+sudo apt-get update
+sudo apt-get install matrix-creator-malos-wakeword --no-install-recommends
+```
+
+Nodejs and npm on RaspberryPi:
+
+``` 
+curl -sL https://deb.nodesource.com/setup_6.x | sudo bash -
+sudo apt-get install nodejs
+```
+
+## Run sample DEMO
+
+Before, copy on your RaspberryPi language models and dictionary for run the DEMO or generated
+them (see section: Custom language and phrases for recognition)
+
+``` javascript
+cd /home/pi
+git clone --recursive https://github.com/matrix-io/matrix-malos-wakeword.git
+cp -r matrix-malos-wakeword/assets .
+```
+
+
+Run nodejs example and say some voice commands: `mia ring red`, `mia ring
+orange`, `mia ring clear` for example:
+
+``` javascript
+cd matrix-malos-wakeword
+git submodule init && git submodule update
+cd src/js_test
+node test_wakeword.js
+```
+
+## Documentation
+
+The driver follows the [MALOS protocol](https://github.com/matrix-io/matrix-creator-malos/blob/master/README.md#protocol).
+
+
+### 0MQ Port
+```
+60001
+```
+
+### Protocol buffers
+
+``` javascript
+message WakeWordParams {
+  // Wake Word
+  string wake_word = 1;
+
+  // Mic channel
+  enum MicChannel {
+    channel0 = 0;
+    channel1 = 1;
+    channel2 = 2;
+    channel3 = 3;
+    channel4 = 4;
+    channel5 = 5;
+    channel6 = 6;
+    channel7 = 7;
+    channel8 = 8;
+  }
+
+  MicChannel channel = 2;
+
+  // language model path from lmtool or similar alternative:
+  // http://www.speech.cs.cmu.edu/tools/lmtool-new.html
+  string lm_path = 3;
+
+  // dictionary path from lmtool 
+  string dic_path = 4;
+
+  // enable pocketsphinx verbose mode
+  bool enable_verbose = 5;
+
+  // stop recognition service
+  bool stop_recognition = 6;
+}
+
+```
+The message is defined in [driver.proto](https://github.com/matrix-io/protocol-buffers/blob/master/malos/driver.proto).
+
+### JavaScript example
+
+Enhanced description of the [sample source code](src/js_test/test_wakeword.js).
+
+First, define the address and port of the MATRIX Creator. In this case we make it be `127.0.0.1`
+because we are connecting from the local host but it needs to be different if we
+connect from another computer.
+
+``` javascript
+var creator_ip = '127.0.0.1'
+var creator_wakeword_base_port = 60001;
+```
+
+#### Config and start wakeupword service
+
+``` javascript
+function startWakeUpRecognition(){
+  console.log('<== config wakeword recognition..')
+  var wakeword_config = new matrixMalosBuilder.WakeWordParams;
+  wakeword_config.set_wake_word("MIA");
+  wakeword_config.set_lm_path("/home/pi/assets/9854.lm");
```

Comment:
is a language models and voice commands dictionary, pre-generated with `lmtool` from PocketSphinx webpage [1], the idea from next release is remove this dependence 

[lmtool](http://www.speech.cs.cmu.edu/tools/lmtool-new.html)